src/explore-education-statistics-admin/src/pages/release/ReleaseStatusPage.tsx

```diff
@@ -113,7 +117,10 @@ const ReleaseStatusPage = () => {
         </SummaryListItem>
         {release.approvalStatus === 'Approved' && (
           <SummaryListItem term="Release process status">
-            <ReleasePublishingStatus releaseId={releaseId} />
+            <ReleasePublishingStatus
+              releaseId={releaseId}
+              refreshPeriod={refreshPeriod}
+            />
```

Comment:
Missing front end change to `ReleaseStageStatuses` in `releaseService.ts` to remove `dataStage`.

Reply:
Done thanks.  Also removed from test util `ReleaseProgressResponse`